runtime/runtime/src/config.rs

```diff
@@ -120,6 +120,50 @@ pub fn total_send_fees(
             },
             DeleteKey(_) => cfg.delete_key_cost.send_fee(sender_is_receiver),
             DeleteAccount(_) => cfg.delete_account_cost.send_fee(sender_is_receiver),
+            Delegate(signed_delegate_action) => {
+                let delegate_cost = cfg.delegate_cost.send_fee(sender_is_receiver);
+                let delegate_action = &signed_delegate_action.delegate_action;
+                let sender_is_receiver = delegate_action.sender_id == delegate_action.receiver_id;
```

Comment:
I think we should not overwrite `sender_is_receiver` here. The `sender_is_receiver` condition only exists to make local receipts cheaper because they don't need to be transferred over the network. Local is defined as being the exact same account because any two accounts could always be resharded to different shards.

This cost here is for paying the send step from Relayer to Alice. So it makes more sense to me to use those two accounts for defining what whether this step is charged as local or remote receipt. This would also mean the second send step can potentially have a different cost from the first but I personally don't see an issue with that.

Reply:
Thanks. I forgot to remove this line.